backends/events/sdl/sdl-events.cpp

```diff
@@ -900,8 +938,10 @@ bool SdlEventSource::remapKey(SDL_Event &ev, Common::Event &event) {
 void SdlEventSource::resetKeyboardEmulation(int16 x_max, int16 y_max) {
 	_km.x_max = x_max;
 	_km.y_max = y_max;
-	_km.delay_time = 25;
+	_km.delay_time = 12;
```

Comment:
This makes the "unit of time" for velocities twice as small, e.g. velocities effectively became twice as fast.

I am fixing this by putting in an extra factor that will make velocities independent of _km.delay (as it should have always been), in the next commit.